ddl/reorg.go

```diff
@@ -152,6 +154,8 @@ func (w *worker) runReorgJob(t *meta.Meta, reorgInfo *reorgInfo, lease time.Dura
 		rowCount, doneHandle := w.reorgCtx.getRowCountAndHandle()
 		// Update a job's RowCount.
 		job.SetRowCount(rowCount)
+		totalCount := getTableTotalCount(w, tblInfo)
+		metrics.AddIndexProgress.Set(float64(rowCount) / float64(totalCount))
```

Comment:
Do we need to consider if the case of row count > total count?

Reply:
Same, the ratio should be bigger than 100%.

Reply:
done.